.eslintrc

```diff
@@ -62,11 +62,19 @@
     "keyword-spacing": 1,
     "indent": [1, 2, { "SwitchCase": 1 }],
     "max-len": 0,
+    "operator-linebreak": 0,
+    "object-curly-newline": 0,
+    "implicit-arrow-linebreak": 0,
+    "prefer-destructuring": 0,
```

Comment:
I wouldn't disable this one. We always prefer to use destructuring.

For example, if I see the following code:
```javascript
const foo = object.foo
```

I will comment to use object destructuring.
```javascript
const { foo } = object
```

What downsides do you see to this rule?

Reply:
Sure, I have a rule, maybe is not an universal standard, but I only destructure when I use more than one field of that object. 

E.g. I have a `prop` `user` with `id` and `name` and I will use just the name:

I don't declare `const { name } = this.props.user`, I just use  `user.name`.

Reply:
Right, that pattern is considered a warning for the rule you mention.
Just like you, I prefer to avoid the additional line that requires.